ff/serving/metadata/main/server.go

```diff
@@ -17,11 +17,12 @@ func main() {
 		logger.Panicw("Failed to listen on port", "Err", err)
 	}
 	grpcServer := grpc.NewServer()
-	serv, err := metadata.NewMetadataServer(logger)
+	server, err := metadata.NewMetadataTypesenseServer(logger, "8181", "localhost", "xyz")
```

Comment:
parameters in newmetadatatypesenseserver are logger, host, port, apikey, but this order is logger, port, host, apikey